modules/git/repo_tag.go

```diff
@@ -239,8 +238,6 @@ func (repo *Repository) GetTags() ([]string, error) {
 		return nil
 	})
 
-	version.Sort(tagNames)
-
 	// Reverse order
 	for i := 0; i < len(tagNames)/2; i++ {
```

Comment:
if we are removing sort above, then is this for loop needed?

Reply:
@techknowlogick the tag list on code-view would be reverse

Reply:
master: it is sorted by versions and reversed listed (the tags witch are newest by version are on top oldest on button)
with this: newest tag is on top oldest at the button
with removing: oldest on the top newest at the button